Dockerfile

```diff
@@ -41,16 +41,23 @@ LABEL maintainer="bgauduch@github"
 ARG PYTHON_MAJOR_VERSION
 RUN apt-get update \
   && apt-get install -y --no-install-recommends \
-    ca-certificates=20190110 \
-    git=1:2.20.1-2+deb10u3 \
-    python3=${PYTHON_MAJOR_VERSION}.3-1 \
-    python3-distutils=${PYTHON_MAJOR_VERSION}.3-1 \
+  ca-certificates=20190110 \
+  git=1:2.20.1-2+deb10u3 \
+  python3=${PYTHON_MAJOR_VERSION}.3-1 \
+  python3-distutils=${PYTHON_MAJOR_VERSION}.3-1 \
```

Comment:
Why did you remove the indentation for the package list ?

Reply:
Good question, certainly a linter issue on my side !
Will revert that back to normal